_data/trabajos.yml

```
@@ -34,6 +34,12 @@ TP1:
   entrega: 2019-10-21
   enunciado_link: /tps/2019_2/tp1_strutil
 
+TP1_2:
+  id: TP1 (Parte 2)
+  publicacion: 2019-09-27
+  entrega: 2019-10-30

```

La consigna dice 28/10. Uno de los dos sitios está mal. (Imagino que este, porque el 28/10 sí es lunes.)

---

Efectivamente, la fecha es el 28/10
```suggestion
  entrega: 2019-10-28
```